crates/solidity/testing/snapshots/cst_output/SourceUnit/end_of_file_trivia/generated/0.8.22-failure.yml

```diff
@@ -0,0 +1,33 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+Source: >
```

Comment:
I think this failure is a regression. Is this expected?

Reply:
Fixed in ee862932f8bd82db9c32e6617a957bab4ac1b258 for now